packages/chat-headless-react/THIRD-PARTY-NOTICES

```diff
@@ -1,707 +1,5 @@
 This file was generated with the generate-license-file npm package!
 https://www.npmjs.com/package/generate-license-file
```

Comment:
this looks wrong, can you try regenerating this?

Reply:
I think the issue above is breaking it. Gonna split out the headless-react changes into another pr.